taxonomies.md

```diff
@@ -8,99 +6,100 @@
 		height="42">
 </div>
 
-# Taxonomias
+# Taxonomies
+
+*Taxonomies* can be created and used for item classification. Examples of common taxonomies are *Gender*, *Countries*, etc.
 
-*Taxonomias* podem ser criadas e usadas para a classificação de itens. Exemplos de taxonomias comuns são a de *Gênero*, *Países*, etc.
+> In *WordPress* language, they are custom taxonomies. See the [WordPress documentation](https://wordpress.org/support/article/taxonomies/) to learn more.
 
-> Na linguagem do *WordPress*, elas são taxonomias personalizadas. Consulte a [documentação do WordPress](https://codex.wordpress.org/pt-br:Taxonomias) para saber mais.
+Each taxonomy has a set of **terms**, for example, gender taxonomy can have terms like drama and comedy.
 
-Cada taxonomia tem um conjunto de **termos**, por exemplo, a taxonomia gênero pode ter termos como drama e comédia.
+The terms may have hierarchy, meaning that when you search for items that have terms with *children* (for example, *Samba*), the results will include items that have any of the terms children. (For example, *Samba de Gafieira* and *Samba Maxixe*).
```

Comment:
I hope the example here is clear for _non-brazilians_, maybe using _Rock_, _Progressive Rock_ and _Metal_ would be more universal :rofl:

Reply:
haha, thought about that, too :)